principles.md

```diff
@@ -3,49 +3,42 @@
 Define a unit of software delivery called a Standard Container.
 The goal of a Standard Container is to encapsulate a software component and all its dependencies in a format that is self-describing and portable, so that any compliant runtime can run it without extra dependencies, regardless of the underlying machine and the contents of the container.
 
-The specification for Standard Containers is straightforward.
-It mostly defines 1) a file format, 2) a set of standard operations, and 3) an execution environment.
+The specification for Standard Containers defines:
 
-A great analogy for this is the shipping container.
-Just like how Standard Containers are a fundamental unit of software delivery, shipping containers are a fundamental unit of physical delivery.
+1. configuration file formats
+2. a set of standard operations
+3. an execution environment.
+
+A great analogy for this is the physical shipping container used by the transportation industry.
+Shipping containers are a fundamental unit of delivery, they can be lifted, stacked, locked, loaded, unloaded and labelled.
+Irrespective of their contents, by standardizing the container itself it allowed for a consistent, more streamlined and efficient set of processes to be defined.
+For software Standard Containers offer similar functionality by being the fundamental, standardized, unit of delivery for a software package.
 
 ## 1. Standard operations
 
-Just like shipping containers, Standard Containers define a set of STANDARD OPERATIONS.
-Shipping containers can be lifted, stacked, locked, loaded, unloaded and labelled.
```

Comment:
I just read this thinking that we had labels/annotations but then realized that we haven't added such yet :-\
I see https://github.com/opencontainers/specs/issues/108 and think we should move forward on such